README.md

```diff
@@ -102,14 +102,14 @@ August 2019
 
 ### Abstract
 
-We present a state-of-the-art high-quality real-time SISR algorithm designed to work with Japanese animation and cartoons that is extremely fast *(~3ms with Vega 64 GPU)*, temporally coherent, simple to implement *(~100 lines of code)*, yet very effective. We find it surprising that this method is not currently used 'en masse', since the intuition leading us to this algorithm is very straightforward.  
-Remarkably, the proposed method does not use any machine-learning or statistical approach, and is tailored to content that puts importance to well defined lines/edges while tolerates a sacrifice of the finer textures. The proposed algorithm can be quickly described as an iterative algorithm that treats color information as a heightmap and 'pushes' pixels towards probable edges using gradient-ascent. This is very likely what learning-based approaches are already doing under the hood (eg. VDSR<sup>**[1]**</sup>, waifu2x<sup>**[2]**</sup>).
+We present a state-of-the-art high-quality real-time SISR algorithm designed to work with Japanese animation and cartoons that are extremely fast *(~3ms with Vega 64 GPU)*, temporally coherent, simple to implement *(~100 lines of code)*, yet very effective. We find it surprising that this method is not currently used 'en masse' since the intuition leading us to this algorithm is very straightforward.  
```

Comment:
Line 105: Lack of commas. It should be something along the lines of "state-of-the-art, high-quality and a real-time SISR algorithms...". I believe multiple algorithms are offered, so it should be in plural form